### ISSUE TITLE
warchief rend blackhand (arena event)

### ISSUE THREAD

**Issue by Rumen61** [azerothcore/azerothcore-wotlk#536]
**Description**: Warchief Rend Blackhand (http://www.wowhead.com/npc=10429/warchief-rend-blackhand) dont spawn after killing  3 trash wave
also these 3 waves dont spawn, only the first wave appears and does not move

**Steps to reproduce the problem**:

1. go to Blackrockspire
2. go to arena
3. pull the boss (http://www.wowhead.com/npc=10429/warchief-rend-blackhand)
**Branch(es)**: master
**AC hash/commit**:  

**Operating system**:  win7

**Modules**:  transmog

[//]: # (This template is for problem reports, for other type of reports edit it accordingly)
[//]: # (If this is a crash report, include the crashlog with https://gist.github.com/)
[//]: # (For fixes containing c++ create a Pull Request)

**Comment by BarbzYHOOL** [comment]
Needs to confirm but I tried to do it as a GM and Rend just went in the back of the room and disappeared in the BWL entrance.

Also the door stays open at all time during the event.

I don't know if it happens similarly within a group of players who cleared all the mobs before.

**Comment by Viste** [comment]
classic content on sunwell almost in all thing are broken so i thin it really as he say

**Comment by StygianTheBest** [comment]
> > classic content on sunwell almost in all thing are broken so i thinл it really as he say

--
Have you tested all of the classic content? Have you submitted these bugs to the issue tracker here?

**Comment by Viste** [comment]
@StygianTheBest, plz don't be so trollish you don't do any contribution and now  you ... ah just die idiot

**Comment by Viste** [comment]
btw Anqhiraj event don't work you statisfied?

**Comment by StygianTheBest** [comment]
@Viste Look kid, if you make bold statements, you need to back them up with verifiable data. Running your trap here and name calling isn't going to make anything better and isn't contributing to the cause. Grow up and start being useful. Thanks!

**Comment by StygianTheBest** [comment]
> btw Anqhiraj event don't work you statisfied?

That's a far cry from your recent comment, "classic content on sunwell almost in all thing are broken" so NO, it's not satisfactory. You've cited ONE event, and you haven't submitted ANY of these to the issue tracker.

**Comment by Viste** [comment]
@StygianTheBest you can see my help in master branch. If i want do something i do and share. And now about you  when you want something you just cry as i see.

**Comment by StygianTheBest** [comment]
@Viste Sure thing Salty! I'll keep posting the bugs as I find them, and you'll keep pretending you're something special. I see very little activity of yours on this particular project, and I certainly don't see a list of issues from you regarding things broken with classic content. I doubt you've even played the classic content when it was classic Panderan boy.

**Comment by Viste** [comment]
sorry but play wow from 2004 develope from 2008. And i don't say i'm special just got lulz from you angry kid

**Comment by StygianTheBest** [comment]
@Viste Impressive. I bet you were a 13 year old Russian beta tester playing on a Australian server. Moddin' since '08 ya'll; the same year WoW launched in Russia. Just stop already.

**Comment by Viste** [comment]
@StygianTheBest ahaha russian is not a north korea nothing can stop play on EN or US servers if you want and pay to blizz for account, you so funny

**Comment by ShinDarth** [comment]
guys stop it now please, let's keep it professional here.

**Comment by Viste** [comment]
@ShinDarth no problem but its funny )

**Comment by StygianTheBest** [comment]
@Viste http://docdreyfus.com/psychologically-speaking/the-need-to-feel-special/

**Comment by StygianTheBest** [comment]
Tested this a few days ago. Here are the issues I noticed.

- Only one spawn group appears.
- The portcullis opens and closes, and the creature text occurs, but no spawn appear.
- The creature text is triggered as if there was a kill and proceeds as normal.
- Rend can be seen running behind the portcullis but disappears into the wall.
- Rend will greet you at the stairs after passing through the gate that leads to Blackwing Lair.
- The gate to the Blackwing Lair instance is always open.
- Nefarian is always pacing upstairs even after killing Rend.
